sycl/doc/extensions/supported/sycl_ext_intel_esimd/README.md

```diff
@@ -4,39 +4,44 @@ OneAPI provides the "Explicit SIMD" SYCL extension (or simply "ESIMD") for
 lower-level Intel GPU programming. It provides APIs closely matching Intel GPU ISA
 yet allows to write explicitly vectorized device code. This helps programmer to
 have more control over the generated code and depend less on compiler
-optimizations. The [specification](sycl_ext_intel_esimd.md),
-[API reference](https://intel.github.io/llvm-docs/doxygen/group__sycl__esimd.html), and
-[working code examples](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/) are available on the Intel DPC++ project's github.
-
-**_NOTE:_** _Some parts of this extension is under active development and APIs in the
-`sycl::ext::intel::experimental::esimd` package are subject to change. There are
-currently a number of [restrictions](#restrictions) specified below._
+optimizations.
 
 ESIMD kernels and functions always require the subgroup size of one, which means
 compiler never does vectorization across work-items in a subgroup. Instead,
-vectorization is expressed explicitly in the code by the programmer. Here is a
-trivial example which adds elements of two arrays and writes the results to the
+vectorization is expressed explicitly in the code by the programmer.
+
+**IMPORTANT NOTE: _Some parts of this extension are under active development. The APIs in the
+`sycl::ext::intel::experimental::esimd` namespace are subject to change or removal._**
+
+Please see the additional resources on the Intel DPC++ project's github:
+
+1) [ESIMD Extension Specification](sycl_ext_intel_esimd.md)
+1) [ESIMD API/doxygen reference](https://intel.github.io/llvm-docs/doxygen/group__sycl__esimd.html)
+1) [ESIMD Emulator](./sycl_ext_intel_esimd_emulator.md)
+1) [Examples](./examples/README.md)
+1) [ESIMD LIT tests - working code examples](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/ESIMD/)
```

Comment:
Would it be better to say `ESIMD end-to-end tests`? I think classic understanding of a LIT test is that it is a compile-only test.

Reply:
Fixed in d6b2ac049189639af1f1a13eceaeb2a38b1235e4